[64] Style the Product Details Page using CSS

kojusola opened the pull request:
## Task: `Styled Product Details Page`
- `The product details page was styled. the implementation is screenshot below:`
![phone-product-page1](https://user-images.githubusercontent.com/66546900/97050254-f2461180-1574-11eb-8d77-3f09cf8717c7.jpg)
![phone-product-page2](https://user-images.githubusercontent.com/66546900/97050259-f5410200-1574-11eb-81dc-c00e6a1bf6fb.jpg)
![phone-product-page3](https://user-images.githubusercontent.com/66546900/97050260-f5d99880-1574-11eb-8ef1-aeff709a5304.jpg)
![phone-product-page4](https://user-images.githubusercontent.com/66546900/97050263-f6722f00-1574-11eb-9dfd-2287d64758df.jpg)
![phone-product-page5](https://user-images.githubusercontent.com/66546900/97050267-f7a35c00-1574-11eb-87a8-61e674b960d9.jpg)
![productpage1](https://user-images.githubusercontent.com/66546900/97050290-0558e180-1575-11eb-9905-c3eaeecbe427.jpg)
![productpage2](https://user-images.githubusercontent.com/66546900/97050296-0853d200-1575-11eb-8c11-1800d0295cea.jpg)
![productpage3](https://user-images.githubusercontent.com/66546900/97050307-0ab62c00-1575-11eb-99fa-8484b1b58af0.jpg)

adaugochi commented on submissions/adeolaFafemi/Css/productPage.css:
```diff
@@ -0,0 +1,538 @@
+.product-figures p, .product-figures .buynow, .product-figures label{
+    font-size: 22px;
+    font-weight: bold;
+    font-family: 'Montserrat', sans-serif;
```
Instead of setting font-family everywhere, why not place it in the `body`

kojusola commented:
Added the font-family to the body class and deleted for every other except for the placeholder because it doesn't reflect except specifically added and for cases I used other fonts.
Thank you.

adaugochi reviewed (approved):
well done @kojusola 

i am impress. keep up with the good work
